### DISCUSSION TITLE
Difficulties to specify Ada language version

### DISCUSSION THREAD

**Discussion by RREE** [alire-project/alire#1784] [Support]
The doc states in the catalog specification at about mid of the page:

`build-switches`: optional table of build profile switches definitions.

And further down:

- `Ada_Version`
 o  `Compiler_Default`
 o `Ada83`
 o `Ada95`
 o `Ada05`
 o `Ada12`
 o `Ada2022`
 o `GNAT_Extensions`

However, if I try any of these I get an error (`gnat1: Cannot find: ada2022`). I had to write

```
[build-switches]
"*".Ada_Version = ["-gnat2022"]
```

**Comment by mosteo** [comment]
I just tried
```
[build-switches]
"*".Ada_Version = "Ada2022"
```
and it worked as expected. Is this what you were attempting?

**Reply by Fabien-Chouteau** [reply]
@RREE as shown here it should be a single string (`"Ada2022"`) not an array of strings (`["Ada2022"]`).
